test/nofiles.test.js

```diff
@@ -0,0 +1,90 @@
+const stylelint = require('stylelint')
+const path = require('path')
+
+const code = `import styled from 'styled-components';
+const StyledTable = styled(StyledTableBase)\`
+color: red;
+\``
+const processor = path.join(__dirname, '../lib/index.js')
+const rules = {
+  indentation: 2
+}
+
+describe('no files', () => {
+  let codeFilename
+  let data
+
+  // NOTE beforeEach() runs _after_ the beforeAll() hooks of the describe() blocks
```

Comment:
Good comment, I actually would've thought it was the other way since beforeAll is nested inside the next block

Reply:
Ha, it is not my invention. I imitated from other test files.

Reply:
Yeah I see haha! Too long since I looked at the test files myself I guess! :)